components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -811,6 +811,12 @@ class CLPConfig(BaseModel):
 
     _os_release_file_path: pathlib.Path = PrivateAttr(default=OS_RELEASE_FILE_PATH)
 
+    @validator("aws_config_directory")
+    def validate_aws_config_directory(cls, value: Optional[pathlib.Path]):
+        if value is None:
+            return None
+        return value.expanduser()
```

Comment:
Another way to write the code is. feel free to accept/reject this and I will approve.
```suggestion
        if value is not None:
            value = value.expanduser()
        return value 
```